ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -102,8 +101,41 @@ checkSnapshotIsolation(k, commit_ts) ==
   THEN
     key_si' = [key_si EXCEPT ![k] = FALSE]
   ELSE
-    UNCHANGED <<key_si>>
-
+    UNCHANGED <<key_si>> 
+
+\* Returns index of $e$ in $seq$ 
+getIndex(seq, e) == CHOOSE n \in DOMAIN seq : seq[n] = e  
+
+\* Returns the write whose $ts$ is largest
```

Comment:
em, the last one in the write sequence must have the latest commit ts

Reply:
We did not verify that the write sequence must be ordered

Reply:
we just guarantee that the `commit` is ordered, but the `rollback` is not guaranteed

Reply:
why?

Reply:
because we delete rollback record earlier, now we don't have this problem